backend/src/main/resources/application.properties

```diff
@@ -8,7 +8,9 @@ quarkus.http.port=8085
 quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=postgres
 quarkus.datasource.password=postgres
-quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
+quarkus.datasource.jdbc.url=jdbc:otel:postgresql://127.0.0.1:5432/notifications
```

Comment:
This won't work with [clowder-quarkus-config-source](https://github.com/RedHatInsights/clowder-quarkus-config-source) out of the box. I'll prepare a PR there to retrieve the DB config without introducing a breaking change.

Reply:
https://github.com/RedHatInsights/clowder-quarkus-config-source/pull/118